resolvers/index.js

```diff
@@ -137,4 +140,31 @@ module.exports = {
     siteRecord.fields.InventoryUpdates = inventoryUpdates;
     return siteRecord;
   },
+  [Tables.PurchaseRequests]: {
+    write: async (purchaseRequestRecord, authRecord) => {
+      // Only allow admin users to review purchase requests
+      if (purchaseRequestRecord.fields.hasOwnProperty("Reviewer")) {
```

Comment:
I realize that this isn't the most robust way to check user permissions (if this field is omitted, anyone could still edit purchase request fields).

I was also considering checking `authRecord.fields.Admin` if `purchaseRequestRecord` has a status that isn't 'Pending'. Thoughts?